vars/dockerExecute.groovy

```diff
@@ -249,8 +249,8 @@ private getDockerOptions(Map dockerEnvVars, Map dockerVolumeBind, def dockerOpti
     ]
     def options = []
     if (dockerEnvVars) {
-        for (String k : dockerEnvVars.keySet()) {
-            options.add("--env ${k}=${dockerEnvVars[k].toString()}")
+        for (def e : dockerEnvVars.entrySet()) {
+            options.add("--env ${e.key}=${escapeBlanks(e.value.toString())}")
```

Comment:
IMO we should use the BashUtils for escaping and remove the method escapeBlanksInValues from this step.

Reply:
@daniel-kurzynski Having a auxiliar method in BashUtils for escaping blanks makes clearly sense -> Such a method is now available.

Regarding:

>[...] and remove the method escapeBlanksInValues from this step

* On the formal level: This`escapeBlanksInValues` is not part of that PR.
* More on the content level: Do you think we should have such a method in the BashUtils? Technically this is of course possible, but wouldn't this be to much for such an auxiliar class?

How important is this from your point of view? Is this just "nice to have" or is having `escapeBlanksInValues` in `BashUtils` important from you point of view in order to be able to approve this PR in order to overcome the issue with not being able to have blanks when calling dockerExecute?